browser/ui/brave_browser_command_controller.cc

```diff
@@ -122,8 +124,18 @@ void BraveBrowserCommandController::InitBraveCommandState() {
 #if BUILDFLAG(ENABLE_TOR)
   UpdateCommandForTor();
 #endif
-  UpdateCommandEnabled(IDC_ADD_NEW_PROFILE, !is_guest_session);
-  UpdateCommandEnabled(IDC_OPEN_GUEST_PROFILE, !is_guest_session);
+  PrefService* service = g_browser_process->local_state();
+  DCHECK(service);
```

Comment:
we are absolutely inconsistent in doing this check in the existing code, but we'd better to check it with `if`

Reply:
Are you saying you want:

    bool add_new_profile_enabled = !is_guest_session;
    bool open_guest_profile_enabled = !is_guest_session;
    if (!is_guest_session) {
      if (PrefService* local_state = g_browser_process->local_state()) {
        add_new_profile_enabled =
          local_state->GetBoolean(prefs::kBrowserAddPersonEnabled);
        open_guest_profile_enabled =
          local_state->GetBoolean(prefs::kBrowserGuestModeEnabled);
      }
    }
    UpdateCommandEnabled(IDC_ADD_NEW_PROFILE, add_new_profile_enabled);
    UpdateCommandEnabled(IDC_OPEN_GUEST_PROFILE, open_guest_profile_enabled);

Reply:
sounds, good, but please clang-format this code

Reply:
I'm not sure what "clang-format" means.  I ran clang-format on the file, but after seeing how much it changed in the file, I cherry picked my best guest as to what you mean.  Namely, I applied the 4-space indentation to the variable assignment statements.  But I did not change

    PrefService* local_state ...

to

    PrefService *local_state ...

Did you want me to change that as well?  Cleaving the asterisk (*) to the variable rather than the type is not the style used in the rest of the file, so I'm guessing here...